src/Akka.Streams.Kafka/Stages/Consumers/Abstract/SubSourceLogic.cs

```diff
@@ -720,8 +699,8 @@ private StageActorRef.Receive MessageHandling() => args =>
 
         public override void PostStop()
         {
+            // TODO: need to try to shut consumer actor down here
```

Comment:
Third bug: if we shut down, we need to make sure we kill off the `KafkaConusmerActor` here - this will help the `IControl` complete shutdown too.

I'll do another PR for this